src/AsmResolver.DotNet/Cloning/MemberClonerListener.cs

```diff
@@ -0,0 +1,18 @@
+namespace AsmResolver.DotNet.Cloning
+{
+    /// <inheritdoc/>
+    public abstract class MemberClonerListener : IMemberClonerListener {
```

Comment:
Brace on new line, you might want to setup `.editorconfig` so that these code style layout inconsistencies don't happen :).

Reply:
oh VS betrayed me again xd